OneLogin.xcodeproj/project.pbxproj

```diff
@@ -2000,7 +2000,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				BASE_URL = mobile.staging.account.gov.uk;
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 2;
+				CURRENT_PROJECT_VERSION = 1;
```

Comment:
what's the reason for this change? 👀 
doesn't seem like it's part of the ticket 🙂

Reply:
Just forgot to revert it 😅. It was just so I could get around not having a credentialIssuer before your PR merged. But will change it back now!